torch/_inductor/fx_passes/quantization.py

```diff
@@ -834,6 +843,10 @@ def qconv_weight_prepack(match: Match, *args, **kwargs):
             if clone_node is not None:
                 graph.erase_node(clone_node)
             graph.erase_node(dequant_per_channel)
+            counters["inductor"]["qconv2d_weight_prepack_matcher_count"] += 1
```

Comment:
is it OK to use this? I have a different util for checking nodes actually, would that work: https://github.com/pytorch/pytorch/blob/main/test/quantization/pt2e/test_quantize_pt2e.py#L394

Reply:
Hi @jerryzh168, thanks for the comments. 

- I think add and check the `counters` is more like the way other inductor UT did as:   https://github.com/pytorch/pytorch/blob/ff35e1e45b9869fb3739469da96ea016a4ab739a/test/inductor/test_split_cat_fx_passes.py#L88-L91 and https://github.com/pytorch/pytorch/blob/8191fb3e060f600090c839d6b86c28dbb687b679/torch/_inductor/fx_passes/split_cat.py#L99

- Per my understanding, I am afraid this function (https://github.com/pytorch/pytorch/blob/main/test/quantization/pt2e/test_quantize_pt2e.py#L394) may not work in this case, since we may not able to get the model after fusion passes but before lowering into CPP Backend.

Reply:
OK sounds good, thanks for the explanation